.gitignore

```diff
@@ -20,7 +20,6 @@
 
 .byebug_history
 .ruby-version
-coverage/
```

Comment:
I think we want it to be ignored otherwise it a load of noise

Reply:
Yup i think we should be able to generate during the build run (after tests run) and store the artefact for viewing. We'll likely need to use the `COVERAGE` env or something if we want some flexibility to toggle. Also  90% is much better than what i was expecting, maybe we should increase the threshold to match instead of the current `75`

Reply:
I agree we should gitignore the coverage files, however with the current test commands in 'build-and-deploy.yml' this does not then generate the coverage directory for the upload coverage task. I have seen it done in other projects, will have a dig around

Reply:
![Screenshot 2022-06-10 at 16 20 28](https://user-images.githubusercontent.com/4527528/173097861-725c68da-20ed-4c0b-9db4-c865d450b9c8.png)